package.json

```diff
@@ -49,5 +50,6 @@
   },
   "lint-staged": {
     "**/*": "prettier --write --ignore-unknown"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Could you please remove this as we are still using `npm` as the pm?

Reply:
Oh man, this is what I get for being used to running `yarn`.  My bad.  I've removed it now.